RootTools/python/samples/samples_Stop4Body.py

```diff
@@ -0,0 +1,83 @@
+SMS_T2tt_genHT_160_genMET_80_mStop_400_mLSP_350   =    kreator.makeMCComponent( "SMS_T2tt_genHT_160_genMET_80_mStop_400_mLSP_350"  ,  "/SMS-T2-4bd_genMET-80_mStop-400_mLSP-350_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/MINIAODSIM", "CMS", ".*root" , xSec=1.835)
+
+signalFullSim= [
+SMS_T1bbbb_mGluino1500_mLSP100,
+SMS_T1tttt_mGluino1500_mLSP100,
+SMS_T2tt_genHT_160_genMET_80_mStop_275_mLSP_205,
+SMS_T2tt_genHT_160_genMET_80_mStop_350_mLSP_330,
+SMS_T2tt_genHT_160_genMET_80_mStop_400_mLSP_350,
+]
+
+
+SMS_TChiWZ_ZToLL  = kreator.makeMCComponent("SMS_TChiWZ_ZToLL", "/SMS-TChiWZ_ZToLL_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/RunIISpring16MiniAODv2-PUSpring16Fast_80X_mcRun2_asymptotic_2016_miniAODv2_v0-v2/MINIAODSIM",  "CMS", ".*root", )
+SMS_TChipmWW      = kreator.makeMCComponent("SMS_TChipmWW"    , "/SMS-TChipmWW_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/RunIISpring16MiniAODv2-PUSpring16Fast_80X_mcRun2_asymptotic_2016_miniAODv2_v0-v1/MINIAODSIM", "CMS", ".*root" )
+
+
+
+SMS_T2tt_dM_10to80_2Lfilter  = kreator.makeMCComponent("SMS_T2tt_dM_10to80_2Lfilter", "/SMS-T2tt_dM-10to80_2Lfilter_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/RunIISpring16MiniAODv2-PUSpring16Fast_80X_mcRun2_asymptotic_2016_miniAODv2_v0-v1/MINIAODSIM" , "CMS", ".*root", )
+
+
+SignalSUSY = [
+SMS_T2tt_dM_10to80_genHT_160_genMET_80,
+SMS_T2tt_dM_10to80_2Lfilter,
+SMS_T2tt_dM_10to80_genHT_160_genMET_80_mWMin_0p1,
+SMS_T2bW_X05_dM_10to80_genHT_160_genMET_80_mWMin_0p1,
+]
+
+### ----------------------------- summary ----------------------------------------
+
+
+signalSamples = SignalSUSY
+
+samples = signalSamples + signalFullSim
+
+### ---------------------------------------------------------------------
+
+from CMGTools.TTHAnalysis.setup.Efficiencies import *
+dataDir = "$CMSSW_BASE/src/CMGTools/TTHAnalysis/data"
+
+#Define splitting
+for comp in signalSamples:
+    comp.isMC = True
+    comp.isData = False
+    comp.isFastSim = True
+    comp.splitFactor = 250 #  if comp.name in [ "WJets", "DY3JetsM50", "DY4JetsM50","W1Jets","W2Jets","W3Jets","W4Jets","TTJetsHad" ] else 100
+    comp.puFileMC=dataDir+"/puProfile_Summer12_53X.root"
+    comp.puFileData=dataDir+"/puProfile_Data12.root"
+    comp.efficiency = eff2012#Define splitting
+
+for comp in signalFullSim:
+    comp.isMC = True
+    comp.isData = False
+    comp.isFastSim = False
+    comp.splitFactor = 250 #  if comp.name in [ "WJets", "DY3JetsM50", "DY4JetsM50","W1Jets","W2Jets","W3Jets","W4Jets","TTJetsHad" ] else 100
+    comp.puFileMC=dataDir+"/puProfile_Summer12_53X.root"
+    comp.puFileData=dataDir+"/puProfile_Data12.root"
+    comp.efficiency = eff2012
```

Comment:
You probably need only to set the isFastSim and the splitFactor.
All the rest should either be obsolete run1 stuff or already be defined in makeMCComponent